docs/_user_guide/best_practices_migrating_from_jupyter.md

```diff
@@ -50,26 +51,27 @@ The biggest hurdle in migrating your process from a JupyterHub notebook to `yen-
 * Write a submit script to run your process on `yen-slurm` using your working environment
 
 
-## Python Virtual Environments
+### Python Virtual Environments
```

Comment:
Maybe we can do ## level so it's the same level as Managing Package Dependencies?

Reply:
done